src/libaktualizr/storage/CMakeLists.txt

```diff
@@ -14,5 +25,9 @@ endif()
 set(HEADERS ${HEADERS} invstorage.h)
 set(SOURCES ${SOURCES} invstorage.cc)
 
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
 add_library(storage OBJECT ${SOURCES})
+add_dependencies(check-format storage)
```

Comment:
This is a bit annoying, it also prevents running clang-tidy without compiling.
A way to avoid this would be to generate a .cc file instead of a header, declare the array of data as extern and link it afterwards.